src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -210,6 +215,33 @@ def generate_codelist_results(self):
             self.dataset, parent_col_title, label_col_title, unique_identifier
         )
 
+    @staticmethod
+    def _get_column_component_info_for_output(
+        list_of_info: List[ColumnComponentInfo],
+    ) -> List[Dict[str, Union[str, bool, None]]]:
+        """
+        Returns the column component and column definitions informaiton ready for outputing into a table.
+        """
+
+        return [
+            {
+                "Title": x.column_definition.title,
+                "Type": x.component_type.name,
+                "Required": x.column_definition.required,
+                "Property URL": x.column_definition.property_url,
+                "Observations Column Titles": ""
+                if x.component is None
+                else ", ".join(
+                    [
+                        c.title
+                        for c in x.component.used_by_observed_value_columns
+                        if c.title is not None
+                    ]
+                ),
+            }
+            for x in list_of_info
```

Comment:
Can you call it something other than `list_of_info`? `column_component_infos` may be better, even though I hate `infos`.